scripts/conversion_script.py

```diff
@@ -83,6 +83,30 @@ def to_dict(self):
         }
 
 
+def get_rhel_version():
+    """Currently we execute the task only for RHEL 7 or 8"""
+    print("Checking OS distribution and version ID ...")
+    try:
+        distribution_id = None
+        version_id = None
+        with open("/etc/os-release", "r") as os_release_file:
+            for line in os_release_file:
+                if line.startswith("ID="):
+                    distribution_id = line.split("=")[1].strip().strip('"')
+                elif line.startswith("VERSION_ID="):
+                    version_id = line.split("=")[1].strip().strip('"')
+    except IOError:
+        print("Couldn't read /etc/os-release")
+    return distribution_id, version_id
+
+
+def is_non_eligible_releases(release):
+    eligible_releases = ["7.9"]
+    major_version, minor = release.split(".") if release is not None else (None, None)
+    version_str = major_version + "." + minor
+    return release is None or version_str not in eligible_releases
```

Comment:
I am not sure this will work for centos. I think we just have to check for `"7"`, this is the output on my beaker centos machine

```
[root@kvm-01-guest19 ~]# cat /etc/os-release 
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

This function could probably be removed and in main we can just check that `if dist != "centos" and version != "7":`

Or if we need to check specifically for `7.9` We could maybe use different file?
```
[root@kvm-01-guest19 ~]# cat /etc/centos-release
CentOS Linux release 7.9.2009 (Core)
```

Reply:
Ouch, you're right! I will simplify and look only for 7 in the matching version.

Reply:
Looks alright now!